modules/tinymce/src/themes/silver/main/ts/Events.ts

```diff
@@ -6,20 +6,23 @@ import Editor from 'tinymce/core/api/Editor';
 import { AfterProgressStateEvent } from 'tinymce/core/api/EventTypes';
 import { EditorEvent } from 'tinymce/core/api/util/EventDispatcher';
 
-const setup = (editor: Editor, mothership: Gui.GuiSystem, uiMothership: Gui.GuiSystem): void => {
+const setup = (editor: Editor, mothership: Gui.GuiSystem, uiMotherships: Gui.GuiSystem[]): void => {
```

Comment:
Is there a reason we need to distinct the motherships here. From what I can see it does the same thing to both mothership and the uiMotherships? So one just concats the arrays before calling this or is that confusing.

Reply:
The separation is a bit more conceptual. I don't mind either way, but that's the distinction.